inst/tests/tests.Rraw

```diff
@@ -13284,6 +13284,11 @@ test(1958.19, fread(text=txt, nrows=0L, verbose=TRUE), data.table(A=integer(), B
 txt = paste(c("A,B\n1,2\n", rep("3,4\n",5100), "3,4.1\n", rep("5,6\n",4900)), collapse="")
 test(1958.20, fread(text=txt, nrows=0L, verbose=TRUE), data.table(A=integer(), B=integer()),
               output="Sampled 1049 rows.*at 11 jump points")
+# invalid head position for nrows=0 #5868
+test(1958.21, fread(sprintf('%s%s\n', 'a,b', paste0('\n', sample(c('', '3'), 1e6L, TRUE), sample(c(',a',',b',',c'), 1e6L, TRUE), collapse="")), nrows=0L),
```

Comment:
is such a big table really needed?

Reply:
For these two columns, `n = 599101` (~6e5) is sufficient. Or we add more columns.

Reply:
I think a comment with this info near the test would be good, and 6e5 makes sense to me (already 40% smaller)